cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -0,0 +1,46 @@
+{# ==========================================================================
+
+   form-field-with-button.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a form field with text when given:
+
+   settings:  An object of used to customize the markup.
+
+   settings.field.attributes:  Additional attributes to add the input.
+
+   settings.field.info:        The text information to display above the field.
+
+   settings.field.label:       The label for the input.
+
+   settings.field.name:        The name/id for the input.
+
+   settings.field.placeholder: The placeholder used for the input.
+
+   settings.btn_text:          The text to use for the submit button.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+<div class="m-form-field-with-button">
+    <div class="form-group">
+        <label for="{{ settings.field.name }}">
+            <b>{{ settings.field.label }}</b>
+            {% if settings.field.attributes.find('required') > -1 %}
+                (required)
+            {% endif %}
+        </label>
+        <input id="{{ settings.field.name ~ '_' ~ range(1, 100) | random }}"
+               type="{{ settings.field.type }}"
+               name="{{ settings.field.name }}"
+               placeholder="{{ settings.field.placeholder }}"
+               class="m-form-field-with-button_field"
+               {{ settings.field.attributes or '' }}>
+    </div>
+    {{ settings.field.info or '' }}
+    <input class="btn btn__full" type="submit" value="{{ settings.btn_text }}">
```

Comment:
Or maybe this molecule should be two separate molecules, one for the field and one for the button?